doc/usermanual/CMakeLists.txt

```diff
@@ -281,9 +283,137 @@ if(can_build_usermanual)
 			install(FILES ${pdffile} DESTINATION ./share/doc/darktable/usermanual OPTIONAL)
 		endforeach(language)
 	endif()
+#
+# LUA API GENERATION
+#
+
+	#
+	# draft pdf API
+	#
+	add_custom_command(
+		OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/darktable-api.pdf
+		COMMAND xsltproc --output ${CMAKE_CURRENT_BINARY_DIR}/darktable-api.fo xsl/darktable_fo.xsl  ${CMAKE_CURRENT_SOURCE_DIR}/lua/lua_api.xml
+		COMMAND fop -c fopconfig.xml ${CMAKE_CURRENT_BINARY_DIR}/darktable-api.fo -pdf ${CMAKE_CURRENT_BINARY_DIR}/darktable-api.pdf
+		WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+		DEPENDS  ${CMAKE_CURRENT_SOURCE_DIR}/lua/lua_api.xml
+		COMMENT "Building API" VERBATIM
+	)
+	add_custom_target(darktable-api DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/darktable-api.pdf)
+	add_dependencies(darktable-api target_media_images )
```

Comment:
no, but most people will build both the api and the UM, so no harm... and i might add images at some point, so I might as well have the infra in place